athena-redis/src/main/java/com/amazonaws/athena/connectors/redis/integ/RedisIntegTestHandler.java

```diff
@@ -47,6 +47,8 @@
 
     private static final Logger logger = LoggerFactory.getLogger(RedisIntegTestHandler.class);
 
+    private static final String STANDALONE_REDIS_DB_NUMBER = "10";
```

Comment:
What exactly happens if you specify a redis db number in clustered mode?
1. If it's an error case for redis, athena should just pass that upstream
2. If it's a silent failure, do we warn the user or is it caveat emptor?
3. (Most importantly) should we add tests for it and handle it, or is it "undefined behavior" and we leave it alone?

Reply:
Absolutely nothing happens. Originally I didn't know cluster wasn't supported for db number, and during integ testing I was creating the cluster connection with a db number when writing test data, but the read connection without db number was still able to read the data. So theres. a notice in redis readme, that this flag only applies to standalone. But no error occurs if it was provided for a clustered instance